src/sdk/services/oauthService.ts

```diff
@@ -24,11 +22,28 @@ class OAuthService {
     return originUrl.origin.replace(regex[1], 'accounts');
   };
 
-  private getOAuthAuthorizeUrl = (api: ManifestApi) => {
+  private getOAuthAuthorizeUrl = (api: ManifestApi, redirectUri?: string) => {
     const host = this.getOAuthHost();
-    const scopes = encodeURIComponent(api.scopes.join(' '));
-    const redirectUri = encodeURIComponent(api.redirectUri);
-    return `${host}/oauth/authorize?client_id=${api.applicationId}&redirect_uri=${redirectUri}&response_type=code&scope=${scopes}`;
+    const scopes = encodeURIComponent(api.scopes.join(','));
```

Comment:
Isn't this a breaking change? Or it was never used so far?

Reply:
I don't know why there was a space here. I checked with accounts service, they actually use `+` to concatenate scopes, so comma was wrong. Thanks for bringing this into attention.

Reply:
No it was not, space and plus are the same character in URL.

Reply:
Wow, I didn't know that. Let me revert it to how it was then.

Reply:
Actually [it's more complicated](https://stackoverflow.com/a/47188851). 
TL;DR: The plus should work, but using space was actually more correct according to [RFC3986](https://www.rfc-editor.org/rfc/rfc3986)

Reply:
I can still see comma in code, but the comment is resolved. You said you're going to revert it, i.e. space will be used.

Reply:
I have reverted it.